app/schedulers/background_scheduler.py

```diff
@@ -11,9 +11,17 @@ def init_scheduler():
                       trigger='cron', hour=23, minute=59, second=0, day='*', timezone='Etc/UTC',
                       replace_existing=True)
 
+    scheduler.add_job(id='remove_unverified_cron', func=complete_remove_unverified_users_job,
+                      trigger='cron', hour=23, minute=59, second=0, day='*', timezone='Etc/UTC',
+                      replace_existing=True)
+
     # for tests purposes
     # scheduler.add_job(id='complete_mentorship_relations_cron', func=complete_overdue_mentorship_relations_job,
     #                   trigger='interval', seconds=4,
     #                   replace_existing=True)
 
+    # scheduler.add_job(id='remove_unverified_cron', func=complete_remove_unverified_users_job,
```

Comment:
do we need this comment?

Reply:
Hi Anna! This part is for test purposes, similar to the one above. I've added the explanation in the PR

Reply:
The cron job runs every 24 hours. But for testing purposes the time period is 4 seconds. This is the commented code.